app/views/events/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for @event, html: {class: 'form-horizontal p-4 bg-theme text-white'} do |f|%>
+  <div class = 'form-group'>
+    <%= f.label :title%>
+    <%= f.text_field :title, placeholder: "Event Title", class: 'form-control'%>
+  </div>
+
+  <div class = 'form-group'>
+    <%= f.label :description%>
+    <%= f.text_area :description, placeholder: "Event Description",  class: 'form-control',  cols: "100", rows: "10"%>
+  </div>
+
+  <div>
+    <%= f.date_field :date, class: 'form-control datepicker w-25 mb-3'  %>
+  </div>
+
+  <%= f.submit 'Create Event', class: 'btn btn-success'%>
+  <div>
+    <% if @event.errors.any?%>
+    <div class="alert alert-danger">
+      <% @event.errors.full_messages.each do |error|%>
+      <p class="larger-text"><%= error%></p>
+      <%end%>
```

Comment:
:bulb: No application logic in the views, please. You can create helper methods to get the same benefit.

Reply:
Corrected